contrib/ledger-gen/src/main.rs

```diff
@@ -53,5 +54,6 @@ fn main() {
     let account_data = vec![0u8; 4];
 
     // ----------------------- ONLY CHANGE BELOW THIS LINE -----------------------
-    ledgers::bpf_loader_ledger(&rpc_client, &arc_client, &payer, &program_data, &account_data);
+    // ledgers::bpf_loader_ledger(&rpc_client, &arc_client, &payer, &program_data, &account_data);
+    ledgers::stake_ledger(&rpc_client, &payer);
```

Comment:
Instead of changing this every time on a one-off basis, we should have a deterministic script we can run to generate all the test ledgers. Then we can just add to this in PRs. I think this file is basically that with some tweaks.